### PR TITLE
Rewrite README to be consistent with other repos

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/Psf7EzPB/204-move-away-from-startupsh

Please see the commit messages for more details. I've kept the
long "schemas" section, since it's a comprehensive description
of what the repo does, with actionable and illustrative info for
each of the (non-)migrated docs it renders.

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,6 @@ Government Frontend is a public-facing app to display the majority of documents
 on the /government part of GOV.UK. It is a replacement for the public-facing
 parts of the [Whitehall](https://github.com/alphagov/whitehall) application.
 
-## Screenshots
-
-![A Case Study](https://raw.githubusercontent.com/alphagov/government-frontend/master/docs/assets/case-study-screenshot.png)
```
Comment (kevindew) [inline README.md:9]: Makes sense - given the screenshot is so text heavy it's quite confusing.

File: README.md [hunk 2]
```diff
@@ -51,13 +51,6 @@ Configuration for the navigation links shown on the B variant of the ContentPage
 This is a Ruby on Rails application that fetches documents from
 [content-store](https://github.com/alphagov/content-store) and displays them.
 
-### Dependencies
-
-- [content-store](https://github.com/alphagov/content-store) - provides documents
-- [static](https://github.com/alphagov/static) - provides shared GOV.UK assets and templates.
-- [phantomjs](http://phantomjs.org/) Used by poltergeist for integration testing
```
Comment (kevindew) [inline README.md:62]: I imagine this phantomJS reference was out of date anyway - at least I hope so 🤞

File: README.md [hunk 3]
```diff
@@ -36,48 +34,19 @@ Not all schemas that this app can handle are rendered by it in production.
...
-Or to specify the location explicitly:
-
-`GOVUK_CONTENT_SCHEMAS_PATH=/some/dir/govuk-content-schemas bundle exec rake`
-
-#### Debugging Integration tests
-
-If you want to see the page that is being tested in our integration tests, you can use
-`save_and_open_page` to see what's rendered. This is helpful when a page is mostly comprised of
-GOV.UK Publishing Components
+If you want to see the page that is being tested in our integration tests, you can call `save_and_open_page` to see what's rendered.
```
Comment (kevindew) [inline README.md:49]: Do we know if this tip still works when running the tests in govuk-docker? 

It seems a little inconsistent to have this anyway since it's just a capybara feature and this tip seems to miss most of the info you'd need to actually utilise it.

File: README.md [hunk 4]
```diff
@@ -86,10 +74,20 @@ Or to specify the location explicitly:
...
 `save_and_open_page` to see what's rendered. This is helpful when a page is mostly comprised of
 GOV.UK Publishing Components
 
+### Components
+
+Pages are rendered using components. Components can be specific to government-frontend or shared between applications.
+
+Components specific to government-frontend are [within the application](https://github.com/alphagov/government-frontend/tree/master/app/views/components) and follow rules set out by the [govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components) gem. They are documented in the [government-frontend component guide](https://government-frontend.herokuapp.com/component-guide).
+
+Configuration for the navigation links shown on the B variant of the ContentPagesNav A/B test is covered [separately](docs/navigation-links.md)
```
Comment (kevindew) [inline README.md:57]: This links to a 404 and seems unrelated to components - I'd suggest removing.

File: README.md [hunk 5]
```diff
@@ -86,10 +74,20 @@ Or to specify the location explicitly:
...
+
+
```
Comment (kevindew) [inline README.md:59]: Nit: extra line

**Review by kevindew — COMMENTED** [review]
This looks good, nice work. Just a few minor comments.

File: README.md (hunk 3 above)
Comment (benthorner) [reply README.md]: True, it doesn't. I've removed this in https://github.com/alphagov/government-frontend/pull/2108/commits/7c1903ef8cc1cc332237bc341daf29f1dee070de.

File: README.md (hunk 4 above)
Comment (benthorner) [reply README.md]: Good spot, I've removed it in https://github.com/alphagov/government-frontend/pull/2108/commits/a79d31dccf199d58096c0dccd744ce3ef616be67.

File: README.md (hunk 5 above)
Comment (benthorner) [reply README.md]: 👍, covered by ☝️.

**Comment by benthorner** [comment]
Thanks for the deeper checking @kevindew 💯. I think this one is ready for another look now.

**Review by kevindew — APPROVED** [review]
Nice one
